src/peers/internal.py

```diff
@@ -28,5 +28,7 @@ def handle_action(self, target_type: str, cmd: str, assets: List[Asset], params:
         """Process the given action."""
         if cmd == "print":
             print(params.get("text", "[print command found no text to print]"))
+        elif cmd == "goto-node":
```

Comment:
This feels like it belongs with "next-node" and "prev-node", as a hardcoded command from the UI. Or is there a reason I'm missing to make it an action?

Reply:
I have been thinking that having this command would be nice for quite some time, but I don't really have a good notion of why. It depends on how we do with node traversal and delay I guess. If delayed actions triggers the node traversal immediately even though the actions are carried out with a delay, this can be used to go to a node after a certain time. If node traversal only goes through AFTER all actions have been carried out we get that feature for "free", but then we also get some control latency (or what to call it) with node traversal.

So TL;DR: having this as an action could give us more freedom with delayed actions.

Reply:
(apart from interactions with delay I don't have any examples of when it would be useful. Perhaps events as well?)

Reply:
I would rather that not be possible. Delayed gotos won't be visible in the timeline, and can interrupt the user if they've kept navigating manually. If we have a concrete use-case later, I think it's better to solve it in a way that's more integrated with the timeline.

Reply:
Ok! Will move to UI action instead.